addons/wowl/static/src/services/router.ts

```diff
@@ -11,6 +11,7 @@ export interface Route {
 export interface Router {
   current: Route;
   pushState(hash: Query, replace?: boolean): void;
+  replaceState(hash: Query, replace?: boolean): void;
```

Comment:
@aab-odoo 
@ged-odoo 
This new method in the router may solve our issues:
- Action manager can pushState naturally from within its closure.
- WebClient can catch the event triggered by the action manager to do a replaceState in history and push menu_id and cids